launcher/ui/widgets/ThemeCustomizationWidget.h

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: GPL-3.0-only
+/*
+ *  Prism Launcher - Minecraft Launcher
+ *  Copyright (C) 2022 Tayou <tayou@gmx.net>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, version 3.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include <QWidget>
+#include "translations/TranslationsModel.h"
+
+enum ThemeFields { NONE = 0b0000, ICONS = 0b0001, WIDGETS = 0b0010, CAT = 0b0100 };
+
+namespace Ui {
+class ThemeCustomizationWidget;
+}
+
+class ThemeCustomizationWidget : public QWidget {
+    Q_OBJECT
+
+   public:
+    explicit ThemeCustomizationWidget(QWidget* parent = nullptr);
+    ~ThemeCustomizationWidget();
+
+    void showFeatures(ThemeFields features);
+
+    void applySettings();
+
+    void loadSettings();
+    void retranslate();
+
+   private slots:
+    void applyIconTheme(int index);
+    void applyWidgetTheme(int index);
+    void applyCatTheme(int index);
+
+   signals:
+    int currentIconThemeChanged(int index);
+    int currentWidgetThemeChanged(int index);
+    int currentCatChanged(int index);
+
+   private:
+    Ui::ThemeCustomizationWidget* ui;
+
+    //TODO finish implementing
+    QList<std::pair<QString, QString>> m_iconThemeOptions{ 
+        { "pe_colored",     QObject::tr("Simple (Colored Icons)") }, 
+        { "pe_light",       QObject::tr("Simple (Light Icons)") },     
+        { "pe_dark",        QObject::tr("Simple (Dark Icons)") },
+        { "pe_blue",        QObject::tr("Simple (Blue Icons)") },    
+        { "breeze_light",   QObject::tr("Breeze Light") }, 
+        { "breeze_dark",    QObject::tr("Breeze Dark") },
+        { "OSX",                        "OSX" },        
+        { "iOS",                        "iOS" },          
```

Comment:
also translatable

Reply:
that was not translatable in the original UI file see [`d472776` (#249)](https://github.com/PrismLauncher/PrismLauncher/pull/249/commits/d472776181f348f214c51b72d17519d779213c7e#diff-9e88bef8c982c4c3ccc1e6ae1161932e63a37127566e0bea7971cecb5fcfe73eL85)

![image](https://user-images.githubusercontent.com/31988415/210787347-632c4a55-d2aa-4122-b7e8-760e0575a151.png)
> notr

Reply:
weird. It probably should be, as some languages might have translations for those.

Reply:
I don't think OSX or iOS are translatable

Reply:
> I don't think OSX or iOS are translatable

it is generally good practice to wrap all strings that a user will see in the GUI in `tr()`. There are a lot of lannguages and scripts out there, and while it might still be pronounced the same, it might be transcribed to a different script, other than Latin.

Reply:
though now that i think about it... I think iOS or OSX have translations in e.g. japanese, so I guess I'll make it tr
Probably some weird leftover from MultiMC days.